pkg/common/util/util.go

```diff
@@ -50,6 +51,25 @@ func ConvertPodList(list []corev1.Pod) []*corev1.Pod {
 	return ret
 }
 
+func FilterJobPodList(podlist []corev1.Pod, uid types.UID) []corev1.Pod {
+	if podlist == nil {
+		return nil
+	}
+	ret := make([]corev1.Pod, 0, len(podlist))
```

Comment:
```suggestion
	pods := make([]corev1.Pod, 0, len(podlist))
```

clarify what this function returns.